lua/ttt2/libraries/roles.lua

```diff
@@ -128,6 +128,10 @@ local function SetupData(roleData)
 	-- @realm shared
 	CreateConVar("ttt_" .. roleData.name .. "_traitor_button", tostring(conVarData.traitorButton or 0), SERVER and {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED} or FCVAR_REPLICATED)
 
+	---
+	-- @realm shared
+	CreateConVar("ttt2_ragdoll_pinning_" .. roleData.abbr, tostring(conVarData.ragdollPinning or 0), SERVER and {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED} or FCVAR_REPLICATED)
```

Comment:
```suggestion
	CreateConVar("ttt2_ragdoll_pinning_" .. roleData.name, tostring(conVarData.ragdollPinning or 0), SERVER and {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED} or FCVAR_REPLICATED)
```

Reply:
added, but you missed the other places this is checked / used